context/app/static/js/shared-styles/DialogModal/DialogModal.jsx

```diff
@@ -19,6 +21,7 @@ function DialogModal({
   ...props
 }) {
   const DialogContent = DialogContentComponent || MUIDialogContent;
+  const { selectedRows } = useStore();
```

Comment:
Rather than adding case-specific code to the shared `DialogModal`, what if we add an `error` prop and display the `Alert` if it's not null? This would allow us to handle other error cases the same way in the future and would keep this component more generic/reusable. It would also let you e.g. disable submission of the create workspace form if the `selectedRows.size` condition is met.

Reply:
Done